sections/futures/TradingHistory/TradesHistoryTable.tsx

```diff
@@ -199,6 +200,13 @@ const HistoryContainer = styled.div<{ mobile?: boolean }>`
 		`}
 `;
 
+const TableMainHeader = styled.div`
+	font-size: 13px;
+	color: #ece8e3;
+	padding: 20px 15px;
+	border-bottom: 1px solid rgba(255, 255, 255, 0.12);
```

Comment:
Same for the borer here: `border-bottom: ${(props) => props.theme.colors.selectedTheme.border};`